packages/labs/src/components/tag-input/tagInput.tsx

```diff
@@ -148,13 +161,14 @@ export class TagInput extends AbstractComponent<ITagInputProps, ITagInputState>
 
     public static defaultProps: Partial<ITagInputProps> & object = {
         inputProps: {},
+        inputValue: "",
         separator: ",",
         tagProps: {},
     };
 
     public state: ITagInputState = {
         activeIndex: NONE,
-        inputValue: "",
+        inputValue: this.props.inputValue,
```

Comment:
hmm this is not the implementation i expected. this is a `defaultInputValue`, but what we really want is an actual controlled `inputValue`, which means that updating the prop in the future will be reflected in the state (not just on initial load like you have here).

Reply:
Addressed in https://github.com/palantir/blueprint/pull/1967